Package.swift

```diff
@@ -0,0 +1,27 @@
+// swift-tools-version:5.3
+import PackageDescription
+
+let package = Package(
+    name: "Adapty",
+    platforms: [
+        .iOS(.v9)
+    ],
+    products: [
+        .library(
+            name: "Adapty",
+            targets: ["Adapty"]
+        ),
+    ],
+    dependencies: [
+         .package(url: "https://github.com/krzyzanowskim/CryptoSwift.git", from: "1.3.1"),
```

Comment:
It will be better to bump version up to 1.3.2. We also set dependency to 1.3.2 in Podfile.
```suggestion
         .package(url: "https://github.com/krzyzanowskim/CryptoSwift.git", from: "1.3.2"),
```

Reply:
Fixed

Reply:
Let's use "exact" but not "from" and stick to the exact lib version?

Reply:
Done